source/scss/layout/_main.scss

```diff
@@ -24,6 +24,16 @@ main {
     position: relative;
   }
 
+  .width-100 {
+    width: 100%;
+  }
+}
+
+@include media-breakpoint-up(sm) {
+  .width-25 {
+    flex: 0 0 50%;
+    max-width: 50%;
+  }
 }
 
 @include media-breakpoint-up(md) {
```

Comment:
Media break point inside the class. We want to achieve a better code styling. :)
Also try to refactor it here a bit. It will improve readability and people will know where things are.

Reply:
fixed while adding a class for standard-content